_reports/2023/zhao.md

```diff
@@ -0,0 +1,248 @@
+attacks by 99% or more without creating any problems for honest users.
+
+Gloria has also worked with several other Bitcoin Core contributors,
+including Suhas Daftuar, Pieter Wuille, and Greg Sanders on a ground-up
+redesign of Bitcoin Core's mempool to make it easier to ensure each new
+transaction added to the mempool strictly improves the set of
+transactions available to be mined.  This has long been the goal of
+Bitcoin Core's mempool policy, but Bitcoin Core previously attempted to
+achieve that using a variety of different heuristics.  In the redesigned
+mempool, called [cluster mempool][topic cluster mempool], a fast sorting
+function is used to put every transaction in the mempool into a linear
+order.  If a new transaction is received to a full mempool, it is only
+accepted if it sorts higher than at least one other transaction.  If a
+replace-by-fee ([RBF][topic rbf]) replacement is received, it must rank
+higher than all of the transactions it replaces.
+
+This conceptually simple new design makes it much easier to reason about
+mempool behavior and also provides new capabilities that can be used in
+the development of new policies for avoiding pinning and other problems
+that affect users of contract protocols.  Gloria has been working
+closely with her colleagues to implement, review, and test cluster
+mempool, parts of which have already started being incorporated in
+Bitcoin Core as of early 2024.
+
+Press:
+
+- Sibling replace-by-fee - **Bitcoin Optech Newsletter #287** [¶](https://bitcoinops.org/en/newsletters/2024/01/31/#kindred-replace-by-fee)
+- Discussion with Gloria Zhao, who proposed sibling replace by fee - **Bitcoin Optech Podcast #287** [¶](https://bitcoinops.org/en/podcast/2024/02/01/#kindred-replace-by-fee-transcript)
+
+## Advocacy
+
+> The goal of this work is to get the broader Bitcoin community to
+> understand and participate productively in protocol development. I
+> think most would agree advocacy is important (decentralized open
+> source development is really hard but it’s what we signed up for!).
+> Funding is less of a pain point now than before, but the number of
+> total and new contributors to Bitcoin Core has been steadily
+> decreasing.
+>
+> I limit this to 10-15% of my time, as there is an infinite supply of
+> things to do. A framework I came up with last year was allocating my
+> time across a “funnel.” The top is spreading a message “the code needs
+> work and runs on donations,” and the bottom might be recruiting
+> somebody to work on Bitcoin Core and looking at PRs together.
+
+Gloria came to Brink after being a participant in the [Bitcoin Core Pull
+Request Review Club][pr review club], a project she now co-maintains
+with fellow Brink engineer Stéphan Vuylsteke.  She's also been teaching
+and mentoring new Bitcoin Core developers Abubakar Sadiq Ismail and
+Ishaanam, the later of whom is a high school student that already has 21
+commits merged into Bitcoin Core.
```

Comment:
If you don't mind :) I don't like to namedrop mentees, makes me feel like I am stealing credit from them

```suggestion
and mentoring new Bitcoin Core developers.
```